sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -128,6 +129,15 @@ case class CreateViewCommand(
     qe.assertAnalyzed()
     val analyzedPlan = qe.analyzed
 
+    // CREATE VIEW AS INSERT INTO ... is not allowed, we should throw an AnalysisException.
+    analyzedPlan match {
```

Comment:
This pattern match does not work with multi-inserts (hive feature). Those are represented using a `Union` of inserts.